Automattic/wp-calypso
# Launchpad: Fix site preview box shadow styling

Related to #73324

## Testing / Review Estimation

Test: Short
Review: Short

## Proposed Changes

* Adding fixes for the site preview box-shadow. On production, the site preview iframe and site preview upsell banner have separate shadows. These changes expand on the recent domain upsell banner added to the Launchpad site preview. 

Before:
![CleanShot 2023-02-22 at 13 17 18@2x](https://user-images.githubusercontent.com/10482592/220720129-63defd85-8a4b-4b00-912a-3c30bec8fe82.png)

After:
<img width="750" alt="CleanShot 2023-02-22 at 13 18 23@2x" src="https://user-images.githubusercontent.com/10482592/220720319-c0beb8ba-41c3-4dd9-bb3c-33ce235e6fc0.png">




## Testing Instructions



* Checkout this branch
* Create one launchpad enabled free plan site (we want the upsell domain banner to appear above the site preview) and create a launchpad enabled site with a paid plan. You can also use existing sites.
* Navigate to the launchpad screen
* Compare the launchpad site preview to production (resize the screen, test the device switcher)
* Smoke test other site previews such as the (new site -> write flow) design setup site preview

## Pre-merge Checklist
